requirements-dev.in

```diff
@@ -1,32 +1,19 @@
 -c requirements.txt
-python-language-server
 django-concurrent-test-helper==0.7.0
 django-debug-panel==0.8.3
 django-debug-toolbar==1.9.1
 flake8==3.4.1
 whitenoise
 Pympler
-importmagic
-yapf
-isort
-pyls-isort
-autoflake
-pylint
 mock
 mixer==6.1.3
 pytest
 pytest-django
 pytest-logging
-jedi>=0.9.0
-json-rpc>=1.8.1
-service_factory>=0.1.6
 pytest-pythonpath
 pytest-timeout
 pytest-watch
-rope
-autopep8<=1.4.0
 pre-commit==1.15.1
```

Comment:
`autopep8` is a useful one. I use it frequently to confom py code to pep 8 guidelines that are pre-commit insist on.  `flower` might also be useful for celery task management via web interface. Rest all looks safe to remove in my opinion.

Reply:
I think we're better off doing the autolinting via pre-commit in future (and those don't need to be specified in the dependencies) - just waiting for a good time to resolve #2945 that won't disrupt everyone's work! (especially @ozer550 and you who are doing a lot of work in the backend).

Reply:
Yup that also sounds good. I'm fine with removing those then.

Reply:
I should have removed `flower` in [this PR](https://github.com/learningequality/studio/pull/3332), but `celery` does have plenty of CLI management commands. One simple alternative for monitoring and management is (`cd contentcuration`) `celery -A contentcuration events`